lineage/figures/figureS12.py

```diff
@@ -0,0 +1,106 @@
+"""
+File: figureS12.py
+Purpose: Generates figure S12.
+Figure S12 shows the accuracy of state assignment versus 
+increasing cell numbers for 2-state, 3-state, and 4-state models.
+"""
+import numpy as np
+from ..Analyze import Analyze, Results
+from ..states.StateDistributionGaPhs import StateDistribution as StateDistPhase
+
+from .figureCommon import (
+    getSetup,
+    subplotLabel,
+    commonAnalyze,
+    figureMaker,
+    pi,
+    T,
+    E,
+    min_desired_num_cells,
+    lineage_good_to_analyze,
+    min_num_lineages,
+    max_num_lineages,
+    num_data_points,
+)
+from ..LineageTree import LineageTree
+
+
+def makeFigure():
+    """
+    Makes figure S12.
+    """
+
+    # Get list of axis objects
+    ax, f = getSetup((10, 10), (1, 3))
```

Comment:
(10, 10) gives you a square. This is a 1 by 3 figure, so the width should be ~3 times the length.